content/_data/authors/tomasbjerre.adoc

```diff
@@ -0,0 +1,4 @@
+---
+name: "Tomas Bjerre"
+github: "tomasbjerre"
+---
```

Comment:
Add Bio/Avatar? :)

Reply:
The avatar is automatic if `content/images/avatars/tomasbjerre.jpg` is added right? I added a bio.